[2025-08-19T09:07:39.563+0000] {task_context_logger.py:104} ERROR - Detected zombie job: {'full_filepath': '/opt/airflow/dags/data_audit_dag.py', 'processor_subdir': '/opt/airflow/dags', 'msg': "{'DAG Id': 'data_audit_dag', 'Task Id': 'data_pull_task', 'Run Id': 'manual__2025-08-19T09:00:59.680857+00:00', 'Hostname': '37fc8edea13d', 'External Executor Id': '701262d9-fbd2-4ab7-9b7d-a405a6209496'}", 'simple_task_instance': <airflow.models.taskinstance.SimpleTaskInstance object at 0x74dd71d5a4b0>, 'is_failure_callback': True} (See https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#zombie-undead-tasks)
